test/Microsoft.AspNetCore.Mvc.Razor.Extensions.Test/ModelDirectiveTest.cs

```diff
@@ -14,7 +14,7 @@ namespace Microsoft.AspNetCore.Mvc.Razor.Extensions
     public class ModelDirectiveTest
     {
         [Fact]
-        public void ModelDirective_GetModelType_GetsTypeFromLastWellFormedDirective()
+        public void ModelDirective_GetModelType_GetsTypeFromFirstWellFormedDirective()
```

Comment:
This means that `@model` existing multiple times in a single document results in the first valid `@model` being used. If you have a valid model in an import file that carries over and properly overrides proper `@model` directives due to how our `ModelDirective` code passes over the IR.